File: java/src/main/java/mx/iteso/adapter/word/MSLicense.java
package mx.iteso.adapter.word;

/**
 * License doc.
 */
public class MSLicense {

    /**
     * Key to license.
     */
    private int license = 0;

    /**
     *
     * @param version .
     */
    public MSLicense(final int version) {
        this.license = version;
    }

    /**
     *
     * @return license.
     */
    public  int getLicense() {
        return this.license;
    }

    /**
     *
     * @param newLicense .
     */
    public void setLicense(final int newLicense) {
        this.license = newLicense;
    }

    /**
     *
     * @return if is valid or not.
     */
    public boolean isValid() {
        return this.license > 0;
    }
}

File: csharp/AdapterPractice/AdapterPractice/Adapter/Word/MSLicense.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AdapterPractice.Adapter.Word
{
    public class MSLicense
    {

        private int license = 0;

        public MSLicense(int version)
        {
            this.license = version;
        }

        public int getLicense()
        {
            return this.license;
        }

        public void setLicense(int newLicense)
        {
            this.license = newLicense;
        }

        public bool isValid()
        {
            return this.license > 0;
        }
    }
}
